charts/veritable-ui/values.yaml

```diff
@@ -0,0 +1,701 @@
+## @param externalDatabase.host Database host
+## @param externalDatabase.port Database port number
+## @param externalDatabase.user Non-root username for veritable-ui
+## @param externalDatabase.password Password for the non-root username for veritable-ui
+## @param externalDatabase.database veritable-ui database name
+## @param externalDatabase.create Enable PostgreSQL user and database creation (when using an external db)
+## @param externalDatabase.postgresqlPostgresUser External Database admin username
+## @param externalDatabase.postgresqlPostgresPassword External Database admin password
+## @param externalDatabase.existingSecret Name of an existing secret resource containing the database credentials
+## @param externalDatabase.existingSecretPasswordKey Name of an existing secret key containing the non-root credentials
+## @param externalDatabase.existingSecretPostgresPasswordKey Name of an existing secret key containing the admin credentials
+##
+externalDatabase:
+  host: ""
+  port: 5432
+  user: veritable_service
+  password: ""
+  database: veritable-ui 
+  create: true
+  postgresqlPostgresUser: "postgres"
+  postgresqlPostgresPassword: ""
+  existingSecret: ""
+  existingSecretPasswordKey: ""
+  existingSecretPostgresPasswordKey: ""
+
+## @section SQNC-Node Parameters
+node:
+  ## @param node.enabled Enable SQNC-Node subchart
+  enabled: true
+  ## @param node.nameOverride String to partially override sqnc-node.fullname template (will maintain the release name)
+  ##
+  nameOverride: ""
+  ## @param node.fullnameOverride String to fully override sqnc-node.fullname template
+  ##
+  fullnameOverride: ""
+## @param externalSqncNode.host External SQNC-Node hostname to query
+##
+externalSqncNode:
+  host: ""
+  ## @param externalSqncNode.port External SQNC-Node port to query
+  ##
+  port: ""
+
+## @section SQNC-Ipfs parameters
+## @param externalSqncIpfs.host External SQNC-Ipfs hostname
+##
+externalSqncIpfs:
+  host: ""
+  ## @param externalSqncIpfs.port External SQNC-Ipfs port
+  ##
+  port: ""
+
+## @section SQNC-Identity-Service parameters
+## @param externalSqncIdentity.host External SQNC-Identity-Service hostname
+##
+externalSqncIdentity:
+  host: ""
+  ## @param externalSqncIdentity.port External SQNC-Identity-Service port
+  ##
+  port: ""
+
+
+## @section authentication configuration
+##
+## @param auth.idp.clientId IDP Client ID
+## @param auth.idp.publicURLPrefix public URL prefix
+## @param auth.idp.internalURLPrefix internal URL prefix
+## @param auth.idp.authPath IDP authentication path
+## @param auth.idp.tokenPath IDP token path
+## @param auth.idp.JWKSPath IDP JWKS path
+auth:
+  enabled: true
+  idp: 
+    clientId: "veritable-ui"
+    publicURLPrefix: "http://localhost:3080/realms/veritable/protocol/openid-connect"
+    internalURLPrefix: "http://localhost:3080/realms/veritable/protocol/openid-connect"
+    authPath: "/auth"
+    tokenPath: "/token"
+    JWKSPath: "/certs"
```

Comment:
Move all of this to the new Veritable-ui config section

Reply:
this has been deleted